pkg/kubelet/images/image_manager.go

```diff
@@ -153,7 +151,7 @@ func (m *imageManager) EnsureImageExists(pod *v1.Pod, container *v1.Container, p
 
 		return "", imagePullResult.err.Error(), ErrImagePull
 	}
-	m.logIt(ref, v1.EventTypeNormal, events.PulledImage, logPrefix, fmt.Sprintf("Successfully pulled image %q in %v", container.Image, time.Since(startTime)), klog.Info)
+	m.logIt(ref, v1.EventTypeNormal, events.PulledImage, logPrefix, fmt.Sprintf("Successfully pulled image %q in %v", container.Image, imagePullResult.pullDuration), klog.Info)
```

Comment:
Do we potentially want to include both? I'm a bit worried about losing the "whole chain" time, and the difference between the two could help surface other issues.

something like:

`fmt.Sprintf("Successfully pulled image %q in %v (%v including waiting)", container.Image, imagePullResult.pullDuration, time.Since(startTime))`

Reply:
This would be more accurate. I will update the pr.

Reply:
Updated in https://github.com/kubernetes/kubernetes/pull/111772/commits/dc7be1338e3667a3d77a4bd8942ba4e5187e0f8f.